ark/phenotyping/create_cell_som.R

```diff
@@ -19,7 +19,7 @@ library(FlowSOM)
 # a helper function for computing 99.9%
 percentile_99_9_helper <- function(x) {
     if (quantile(as.numeric(x[x > 0]), 0.999) == 0) {
-        return(quantile(as.numeric(x[x > 0], 0.999), 1))
+        return(quantile(as.numeric(x[x > 0], 1))
```

Comment:
The parentheses for the as.numeric is still in the wrong place. I think what you meant was `quantile(as.numeric(x[x > 0]), 1)`. You can try some dummy code out like x=0:5 and see that this doesn't return what you think it does. Also, since this is essentially a max function, you don't need to do the x[x>0] part. So i think `quantile(as.numeric(x),1)` is fine.

Reply:
Crap, it's been a long day.

Reply:
Haha no problem, looks good to me now.